src/utils/index.ts

```diff
@@ -17,4 +19,66 @@ const showToast = async (message: string) => {
       return toast.present();
```

Comment:
Improve indentation.

Reply:
Indentation improved.
Thank You for reviewing my code.